tests/unit/amo/components/TestScreenShots.js

```diff
@@ -25,15 +25,15 @@ describe('<ScreenShots />', () => {
         title: 'A screenshot',
         src: 'http://img.com/one',
         thumbnail_src: 'http://img.com/1',
-        h: 200,
-        w: 320,
+        h: 1024,
+        w: 1366,
```

Comment:
Would it make sense to export the variables in the component and reuse them here?

Reply:
Not a bad idea! Thanks! 😄